purchase_cancel_qty/models/purchase_order.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def button_approve(self, **kwargs):
```

Comment:
keep the same signature as the original method, ie `button_approve(self, force=False)`

Reply:
My bad, I didn't see this kwarg was added in the base module.